openapi.yaml

```diff
@@ -164,704 +128,833 @@ paths:
-            $ref: '#/definitions/ErrorArray'
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/image:
+        '204':
+          description: 'SSN accepted'
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '409':
+          $ref: '#/responses/409'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/images:
     get:
-      description: |
-        Get an ID card image for a person. If the person exists but doesn't have
-        an ID card image, a default image will be returned.
+      description: Get a person's image
       tags:
         - images
-      operationId: getImage
+      operationId: getImages
       produces:
         - image/jpeg
         - application/json
       parameters:
+        - $ref: '#/parameters/osuId'
         - in: query
           name: width
+          # Maximum of 2000 is chosen due to it being a sufficiently
+          # large size and the response size starts to balloon at this value
+          maximum: 2000
+          minimum: 1
+          default: 100
           description: Resizes image to a pixel value for width while keeping original aspect ratio
           required: false
           type: integer
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/pretty'
-        - $ref: '#/parameters/authorization'
       responses:
-        200:
+        '200':
           description: Successful response
           schema:
             type: file
-          headers:
-              Content-type:
-                type: string
-                description: image/jpeg
-        404:
-          description: Person not found
-          schema:
-            $ref: '#/definitions/Error'
-  /persons/{osuID}/meal-plans:
+        '400':
+          $ref: '#/responses/400'
+        '404':
+          $ref: '#/responses/404'
+        '500':
+          $ref: '#/responses/500'
+  /persons/{osuId}/meal-plans:
     get:
       description: Get meal plans for a person
       tags:
         - meal-plans
       operationId: getMealPlans
       parameters:
-        - $ref: '#/parameters/osuID'
-        - $ref: '#/parameters/authorization'
+        - $ref: '#/parameters/osuId'
+        - name: filter[balance][gt]
+          in: query
+          type: number
+          format: float
+          required: false
+          description: Filter results based on the remaining balance.
+        - name: filter[balance][gte]
+          in: query
+          type: number
+          format: float
+          required: false
+          description: Filter results based on the remaining balance.
+        - name: filter[lastUsedDateTime][gt]
+          in: query
+          type: string
+          format: date-time
+          required: false
+          description: Filter results based on the last date and time the meal-plan was used.
+        - name: filter[lastUsedDateTime][gte]
```

Comment:
Not sure, but I think we can remove `gt` and only keep `gte` and then  add `lte` as operations for these parameters.